streams/src/main/java/org/apache/kafka/streams/kstream/Transformer.java

```diff
@@ -64,7 +64,7 @@
      *
      * @param context the context
      */
-    void init(final ProcessorContext context);
+    void init(final ProcessorContext<Object, Object> context);
```

Comment:
In the KIP, I proposed to leave the existing Processor and Transformer init methods with a raw ProcessorContext argument. This will give all users a rawtypes warning with no recourse. Instead, in this PR, I'm adding `<Object, Object>` bounds instead. This at least allows users to also add the generics if they desire, while not changing the practical bounds at all. If this proves acceptable, I'll update the KIP.

Reply:
Not sure if I understand this. Can you elaborate?

Btw: to we think we will have any path forward to let user specify the actually return type (that is currently only `R` but does not specify the actually key or value type), such that we would pass in `ProcessorContext<K, V>`? Can't remember if we discussed this on the KIP (just a side question).

Reply:
Thanks for clarification.